.github/workflows/ci.yml

```diff
@@ -167,6 +167,27 @@ jobs:
           cd ../
           tflite-micro/tensorflow/lite/micro/tools/ci_build/test_bluepill_renode.sh tflite-micro/
 
+  cortex_m_qemu:
+    runs-on: ubuntu-latest
+
+    name: Cortex-M QEMU Unit Tests (presubmit)
+    steps:
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.trigger-sha }}
+      - name: Install dependencies
+        run: |
+          pip3 install Pillow
+          pip3 install Wave
+          pip3 install numpy
+      - name: Test
+        run: |
+          cd ../
+          tflite-micro/tensorflow/lite/micro/tools/ci_build/test_cortex_m_qemu.sh tflite-micro/
```

Comment:
this will need to use the Docker container, similar to the Code Style (presubmit).